### PR TITLE
Use cinder feature sitemaps, cleanup.

### PR DESCRIPTION
**Pull request by dblock**
Requires https://github.com/artsy/cinder/pull/126.

### CODE REVIEW

**Comment by eessex** [comment]
Looks like the same failure I got earlier, just started a rebuild -- so close to getting these fully converted!

**Comment by dblock** [comment]
Looks 💚  now. Someone should look at that failure :)

**Comment by dblock** [comment]
@eessex This is now ready to merge. Needs a somewhat careful CR since there's quite a bit of code removed/stuff renamed. Thx.

File: desktop/apps/sitemaps/index.coffee [hunk 1]
```diff
@@ -6,17 +6,16 @@ app.set 'views', __dirname + '/templates'
 app.set 'view engine', 'jade'
 
 app.get '/robots.txt', routes.robots
-app.get '/news_sitemap.xml', routes.articles
-app.get '/video_sitemap.xml', routes.video
+app.get '/sitemap-news.xml', routes.news
+app.get '/sitemap-video.xml', routes.video
 app.get '/sitemap-misc.xml', routes.misc
 app.get '/sitemap-cities.xml', routes.cities
-app.get '/sitemap.xml', routes.index
```
Comment (eessex) [inline desktop/apps/sitemaps/index.coffee]: Is the main index nonexistent now?

File: desktop/apps/sitemaps/routes.coffee [hunk 2]
```diff
@@ -24,31 +24,7 @@ sitemapProxy = httpProxy.createProxyServer(target: SITEMAP_BASE_URL)
...
-    return next(err) if err
-    res.set 'Content-Type', 'text/xml'
-    res.render('index', {
-      pretty: true
-      allPages: allPages
-      resources: resources
-    })
+      res.render('news', { pretty: true, articles: recentArticles })
 
 @misc = (req, res, next) ->
```
Comment (eessex) [inline desktop/apps/sitemaps/routes.coffee]: Curious what would fall under the misc. category

File: desktop/apps/sitemaps/routes.coffee [hunk 3]
```diff
@@ -99,15 +65,18 @@ sitemapProxy = httpProxy.createProxyServer(target: SITEMAP_BASE_URL)
...
     Sitemap: #{APP_URL}/sitemap-articles.xml
     Sitemap: #{APP_URL}/sitemap-artists.xml
+    Sitemap: #{APP_URL}/sitemap-features.xml
     Sitemap: #{APP_URL}/sitemap-genes.xml
     Sitemap: #{APP_URL}/sitemap-artworks.xml
     Sitemap: #{APP_URL}/sitemap-images.xml
     Sitemap: #{APP_URL}/sitemap-partners.xml
     Sitemap: #{APP_URL}/sitemap-shows.xml
     Sitemap: #{APP_URL}/sitemap-fairs.xml
+    Sitemap: #{APP_URL}/sitemap-video.xml
```
Comment (eessex) [inline desktop/apps/sitemaps/routes.coffee]: Do we need a line here for news?

File: desktop/apps/sitemaps/index.coffee (hunk 1 above)
Comment (dblock) [inline desktop/apps/sitemaps/index.coffee]: Yup. This is what is left in it:

```xml
<sitemapindex xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
<sitemap>
<loc>https://www.artsy.net/sitemap-misc.xml</loc>
</sitemap>
<sitemap>
<loc>https://www.artsy.net/sitemap-cities.xml</loc>
</sitemap>
<sitemap>
<loc>https://www.artsy.net/sitemap-features-1.xml</loc>
</sitemap>
<sitemap>
<loc>https://www.artsy.net/sitemap-features-2.xml</loc>
</sitemap>
<sitemap>
<loc>https://www.artsy.net/sitemap-fairs-1.xml</loc>
</sitemap>
<sitemap>
<loc>https://www.artsy.net/sitemap-fairs-2.xml</loc>
</sitemap>
<sitemap>
<loc>https://www.artsy.net/sitemap-fairs-3.xml</loc>
</sitemap>
<sitemap>
<loc>https://www.artsy.net/sitemap-fairs-4.xml</loc>
</sitemap>
<sitemap>
<loc>https://www.artsy.net/sitemap-fairs-5.xml</loc>
</sitemap>
<sitemap>
<loc>https://www.artsy.net/sitemap-fairs-6.xml</loc>
</sitemap>
<sitemap>
<loc>https://www.artsy.net/sitemap-fairs-7.xml</loc>
</sitemap>
<sitemap>
<loc>https://www.artsy.net/sitemap-fairs-8.xml</loc>
</sitemap>
</sitemapindex>
```

I moved the ones still in use into robots.txt and fairs and features were rewritten as cinder jobs.

File: desktop/apps/sitemaps/routes.coffee (hunk 2 above)
Comment (dblock) [inline desktop/apps/sitemaps/routes.coffee]: About page, etc., https://www.artsy.net/sitemap-misc.xml

Comment (kanaabe) [inline desktop/apps/sitemaps/routes.coffee]: stuff like about, privacy, terms will be in misc

File: desktop/apps/sitemaps/routes.coffee (hunk 3 above)
Comment (dblock) [inline desktop/apps/sitemaps/routes.coffee]: Sure we do. Good catch. Re-added.

**Comment by kanaabe** [comment]
👍  lgtm!

**Comment by eessex** [comment]
Yes lgtm too!

**Pull request merged**